compiler/Eta/CodeGen/Rts.hs

```diff
@@ -43,7 +43,7 @@ frameType         = obj stackFrame
 conType           = obj stgConstr
 thunkType         = obj stgThunk
 rtsConfigType     = obj rtsConfig
-rtsOptsEnbledType = obj rtsOptsEnbled
+rtsOptsEnabledType = obj rtsOptsEnabled
```

Comment:
I've left this change in. I understand it's changing a token. I'm not sure if it's a public or private token. If it's public, it should be dropped. If it's private, then I expect the `=`s should be re-aligned. Since the odds are in favor of dropping, it wasn't worth it for me to adjust the indentation.